client/src/components/Modal/index.tsx

```diff
@@ -23,16 +23,16 @@ const Modal = (props: IModalProps): JSX.Element => {
 };
 
 const Wrapper = styled.div`
-  ::before {
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 100vw;
-    height: 100vh;
-    background: #000;
-    opacity: 0.3;
-    content: '';
-  }
+  //::before {
```

Comment:
나중에 이것도 제가 일괄적용해서 올려버릴까요🤔

```suggestion
  &::before {
```

Reply:
제꺼에선 아직 적용이 안되었으니, 일단 이렇게 올리고 은성님께서 수정해주셔도 될 거 같아요!
아마 충돌 해결하시면서 정리될 거 같긴 합니다 ㅎㅎ

Reply:
아 잘못 이해했네요.
그냥 제가 수정해서 올리겠슴당! ㅎㅎ